style.css

```diff
@@ -516,6 +468,84 @@ textarea {
   margin-left: 30px;
 }
 
+.links {
```

Comment:
- [OPTIONAL] Great job so far :raised_hands: but I think it would be better if you could use a `position: fixed;`  instead of absolute. Fixed position prevents your navigation page menu page from scrolling. The navigation is always fixed to the page instead of scrolling. Please change this. This will make your website similar to the original design. 💯 🥇 


- [OPTIONAL] What about preventing the sub-background from scrolling when the navbar is open? You can use a CSS property `over-flow: hidden;` to do that, which will make your mobile menu correspond to the original design.💯 🥇 

 - [ OPTIONAL] Nice implementing the design, however, it would be better if you add a little bit of blur to your overlay in order to match the Figma design and also make a good contrast 🎨 👨🏻‍🎨 





-  Adding these features will make your design look great and look exactly like the Figma design, I highly recommend you do this in the meantime, I'll approve this project since the previous reviewer is skipping this.


| Your design | Figma design |
| --- | --- |
| ![tresbad](https://user-images.githubusercontent.com/79058364/174133622-11c7aff6-13ab-427e-ad60-75136954d497.png)| ![image](https://user-images.githubusercontent.com/61790587/167505757-7e8c3eba-c166-4f42-842b-829d676cabd5.png) |